tmail-backend/mailets/src/test/java/com/linagora/tmail/mailet/dns/DkimDnsValidatorTest.java

```diff
@@ -0,0 +1,111 @@
+/********************************************************************
+ *  As a subpart of Twake Mail, this file is edited by Linagora.    *
+ *                                                                  *
+ *  https://twake-mail.com/                                         *
+ *  https://linagora.com                                            *
+ *                                                                  *
+ *  This file is subject to The Affero Gnu Public License           *
+ *  version 3.                                                      *
+ *                                                                  *
+ *  https://www.gnu.org/licenses/agpl-3.0.en.html                   *
+ *                                                                  *
+ *  This program is distributed in the hope that it will be         *
+ *  useful, but WITHOUT ANY WARRANTY; without even the implied      *
+ *  warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR         *
+ *  PURPOSE. See the GNU Affero General Public License for          *
+ *  more details.                                                   *
+ ********************************************************************/
+
+package com.linagora.tmail.mailet.dns;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Optional;
+
+import org.apache.james.dnsservice.api.DNSService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class DkimDnsValidatorTest {
+
+    private DNSService dnsService;
+    private DkimDnsValidator validator;
+
+    @BeforeEach
+    void setUp() {
+        dnsService = mock(DNSService.class);
```

Comment:
Do not use mock (!)

Prefer using conveience InMemoryDNSService please.

Reply:
Still using mock?